content/5_Plat/Centroid.mdx

```diff
@@ -190,43 +192,51 @@ subgraphs.
 
 General centroid code is shown below.
 
-<IncompleteSection>
-
-Ben - this is not easy to understand :/
-
-</IncompleteSection>
-
 <LanguageSection>
 
 <CPPSection>
 
 <!-- pulled from https://codeforces.com/contest/1303/submission/76216413, which I think is my most recent centroid problem -->
 
 ```cpp
-bool r[MN];  // removed
-int s[MN];   // subtree size
-int dfs(int n, int p = 0) {
-	s[n] = 1;
-	for (int x : a[n])
-		if (x != p && !r[x]) s[n] += dfs(x, n);
-	return s[n];
+vector<int> adj[maxn];  // adjacency list
+bool r[MN];             // removed
+int s[MN];              // subtree size
+
+int dfs(int u, int p = 0) {
+	s[u] = 1;
+	for (int v : adj[u]) {
+		if (v != p && !r[v]) s[u] += dfs(v, u);
+	}
+	return s[u];
 }
-int get_centroid(int n, int ms,
-                 int p = 0)  // n = node, ms = size of tree, p = parent
-{
-	for (int x : a[n])
-		if (x != p && !r[x])
-			if (s[x] * 2 > ms) return get_centroid(x, ms, n);
-	return n;
+
+/*
+ * This function returns the centroid of subtree
+ * containing node u after node removals saved in r
+ * u -> current node
+ * sz -> size of current subtree after node removals
+ * p -> parent of u
+ */
```

Comment:
(optional)

standardize function description?

Reply:
I could not find what is the standard function description used in this repo, can you help me on this?

Reply:
just look around for the tokens @param or @return, it should be obvious

Reply:
Ok, thanks